numpy/core/fromnumeric.py

```diff
@@ -3279,7 +3279,9 @@ def around(a, decimals=0, out=None):
 
     Notes
     -----
-    `~numpy.round` is often used as an alias for `~numpy.around`.
+    `~numpy.round` and `~np.round_` are disrecommended 
```

Comment:
I confess this is a confusing subject :) There is a `round` defined in `numpy/array_api/_elementwise_functions.py`, apparently it will be part of the common array_api. @asmeurer Is that the case? And should we then recommend it as portable? I think that is the main concern here.

Note that there is also an `ndarray.round` method, and `round` is used a lot in the Notes section of `around`. For those interested, `round` itself is defined in `numpy/ma/core.py` and `numpy/core/__init__.py`

Reply:
The functions in array_api shouldn't be recommended as they only work on array_api arrays. But we should recommend the functions that are part of the spec, such as `round`, as other libraries will be using those.

Reply:
Maybe we should mention spec functions in the See Also section? This needs some discussion. @rgommers Thoughts?

Reply:
@charris, thank you for the follow up! Should I wait with removing round_ from the autosummary in routines.math.rst then?

Reply:
Documenting `array_api` in general is an open question. It also relates to the still somewhat open question of whether `numpy.array_api` even belongs in NumPy at all (it's still tagged as experimental). I think it's better to discuss it on the mailing list, or one of the dev calls (if you ever want me to join one of the dev calls just let me know beforehand).

Reply:
@inessa I think Ross' suggestion below is a good one. Short version: folks should not use `np.round_`, but `np.round` is fine.  At some point we may want to _prefer_ `np.round`, but that should wait on a discussion as to how we want to handle functions common to numpy and the array_api.

Reply:
I agree with the outcome here, and with what @rossbar and @asmeurer say.

> Maybe we should mention spec functions in the See Also section? This needs some discussion. @rgommers Thoughts?

I'd say no, we should just aim for those names to be the defaults. No need to reference or cross-link to the standard API docs in individual docstrings. Aaron already wrote a separate narrative docs section which details all the differences. Let's just shrink that over time.